src/doc/en/thematic_tutorials/geometry/polyhedra_tutorial.rst

```diff
@@ -199,11 +199,12 @@ It is also possible to define a polyhedron over algebraic numbers.
 
 ::
 
-    sage: sqrt_2 = AA(2)^(1/2)                                                # optional - sage.rings.number_field
-    sage: cbrt_2 = AA(2)^(1/3)                                                # optional - sage.rings.number_field
-    sage: timeit('Polyhedron(vertices = [[sqrt_2, 0], [0, cbrt_2]])')         # optional - sage.rings.number_field  # random
+    sage: # needs sage.rings.number_field
+    sage: sqrt_2 = AA(2)^(1/2)
+    sage: cbrt_2 = AA(2)^(1/3)
+    sage: timeit('Polyhedron(vertices=[[sqrt_2, 0], [0, cbrt_2]])')
```

Comment:
You must put back the tag `# random`. The CI reports a doctest error:
```
sage -t --long --random-seed=286735480429121101562228604801325644303 doc/en/thematic_tutorials/geometry/polyhedra_tutorial.rst
**********************************************************************
File "doc/en/thematic_tutorials/geometry/polyhedra_tutorial.rst", line 205, in doc.en.thematic_tutorials.geometry.polyhedra_tutorial
Failed example:
    timeit('Polyhedron(vertices=[[sqrt_2, 0], [0, cbrt_2]])')
Expected:
    5 loops, best of 3: 43.2 ms per loop
Got:
    25 loops, best of 3: 21.9 ms per loop
**********************************************************************
1 item had failures:
   1 of 100 in doc.en.thematic_tutorials.geometry.polyhedra_tutorial
    [99 tests, 1 failure, 16.62 s]
```

Reply:
Thanks for catching this, fixed in f43552ef2b4